Pipfile

```diff
@@ -0,0 +1,18 @@
+[[source]]
+name = "pypi"
+url = "https://pypi.org/simple"
+verify_ssl = true
+
+[dev-packages]
+pylint = "*"
+black = "*"
+
+[packages]
```

Comment:
Should vision packages be in this? Is this a replacement for requirements.txt?

Reply:
They will need to be at some point, anything required for competition